pkg/controller/propagator/propagation.go

```diff
@@ -639,6 +640,14 @@ func (r *ReconcilePolicy) processTemplates(replicatedPlc *policiesv1.Policy, dec
 	reqLogger := log.WithValues("Policy-Namespace", rootPlc.GetNamespace(), "Policy-Name", rootPlc.GetName(), "Managed-Cluster", decision.ClusterName)
 	reqLogger.Info("Processing Templates..")
 
+	//clear the trigger-update annotation, its only for the root policy shouldnt be in  replicated policies
+	//as it will cause an unnecessary update to the managed clusters
+	annotations := replicatedPlc.GetAnnotations()
+	if _, ok := annotations["policy.open-cluster-management.io/trigger-update"]; ok {
+		delete(annotations, "policy.open-cluster-management.io/trigger-update")
+		replicatedPlc.SetAnnotations(annotations)
```

Comment:
This happens before `CompareSpecAndAnnotation` is called in `handleDecision`, so the root policy will have this annotation but the replicated policy will not, so I think it would always trigger an update to occur regardless of if the spec has changed or not.

Reply:
@mprahl - yes, this does need to happen before compare ? i dont think i understand your concern with it ?

Reply:
I misread the code. This isn't an issue.